trajopt_sco/src/qpoases_interface.cpp

```diff
@@ -165,7 +165,7 @@ void qpOASESModel::update()
     int inew = 0;
     for (unsigned iold = 0; iold < cnts_.size(); ++iold)
     {
-      const Cnt& cnt = cnts_[iold];
+      Cnt& cnt = cnts_[iold];
```

Comment:
Did the const need to be removed?

Reply:
This file did not compile as it was. I am not sure it compiles on Linux currently, I have only tested on Windows.

Reply:
@johnwason can you make the line 66 change?

Reply:
Sorry I can't look at it for a few days